processor/attributesprocessor/attributes_test.go

```diff
@@ -854,340 +854,6 @@ func TestAttributes_Ordering(t *testing.T) {
 	}
 }
 
-func TestAttributes_Matching_False(t *testing.T) {
```

Comment:
Since these tests are for the matching logic - moved this to processor/common/common_test.go